grails-app/controllers/org/pih/warehouse/api/DashboardApiController.groovy

```diff
@@ -214,4 +215,16 @@ class DashboardApiController {
         NumberData numberData = numberDataService.getOpenPurchaseOrdersCount(params)
         render(numberData as JSON)
     }
+
+    def getBackdatedOutboundShipments() {
+        Integer monthsLimit = (params.querySize as Integer) ?: 6
```

Comment:
Please put this 6 month default in a constant. Or even better as a config variable.

Reply:
Also, do we need to define a server-side max? Can the server properly handle the request if we get sent something like `querySize=9999`?